src/Makefile.am

```diff
@@ -246,6 +246,22 @@ libeos_shell_js_la_CPPFLAGS =		\
 
 ########################################
 
+# Set up headers for libeos-shell-fx
+eos_shell_fxincludedir = $(includedir)
+eos_shell_fxinclude_HEADERS = wobbly-effect.h
+
+eos_shell_fx_cppflags = $(AM_CPPFLAGS) $(EOS_SHELL_FX_CFLAGS) -std=c++11 -Werror -Wall -Wextra -Wno-error=unused-parameter -Wno-error=deprecated-declarations
```

Comment:
> given that upstream now does just that -- https://git.gnome.org/browse/gnome-shell/tree/configure.ac#n225

Looks like upstream is making `-Werror` a user option in that case.

Given that I don't think this part of the code is going to change much, I suppose we can remove the compiler flags. We gain consistency and aren't likely to pay any cost on maintainability.